src/tokens/solana.tokenlist.json

```diff
@@ -149826,6 +149826,19 @@
       "extensions": {
         "website": "https://www.dariusdao.com/"
       }
+    },
+    {
+      "chainId": 101,
+      "address": "4oog3cJgry8qrX2KK4nkbq2jZD3vcCmdwpoFykC2fcDW",
+      "symbol": "milox",
+      "name": "MiloX Token",
+      "decimals": 9,
+      "logoURI": "https://github.com/miloxOne/miloxtoken/blob/main/miloxpurple.png",
```

Comment:
Need to update logoURI address to https://raw.githubusercontent.com/miloxOne/miloxtoken/main/miloxpurple.png